improver_tests/psychrometric_calculations/test_PhaseChangeLevel.py

```diff
@@ -572,6 +573,53 @@ def test_snow_sleet_phase_change_reorder_cubes(self):
         self.assertEqual(result.units, Unit("m"))
         self.assertArrayAlmostEqual(result.data, self.expected_snow_sleet)
 
+    def test_snow_sleet_phase_change_zero_radius(self):
```

Comment:
Would it be worth modifying these new tests, so that the expected result when using `grid_point_radius=0` is different to when using `grid_point_radius=2`, as shown in `test_snow_sleet_phase_change`?